packages/amplify-util-uibuilder/src/__tests__/generateComponents.test.ts

```diff
@@ -14,11 +15,17 @@ jest.mock('../commands/utils/featureFlags', () => ({
   ...jest.requireActual('../commands/utils/featureFlags'),
   getTransformerVersion: jest.fn(),
 }));
+jest.mock('amplify-codegen', () => ({
+  ...jest.requireActual('amplify-codegen'),
+  getCodegenConfig: jest.fn(),
+}));
+
 const awsMock = aws as any;
 const utilsMock = utils as any;
+const isDataStoreEnabledMocked = isDataStoreEnabled as any;
+const getTransformerVersionMocked = getTransformerVersion as any;
+const getCodegenConfigMocked = getCodegenConfig as any;
 
-const isDataStoreEnabledMocked = jest.mocked(isDataStoreEnabled);
-const getTransformerVersionMocked = jest.mocked(getTransformerVersion);
```

Comment:
Local tooling issue, I was getting errors that `jest.mocked()` does not exist and the entire test file would be failed. They're replaced with casts to `any` a couple lines higher, not sure why the diff ended up being removals and additions instead of as line updates.